### DISCUSSION TITLE
缠论代码问题求助

### DISCUSSION THREAD

**Discussion by pyxm1618** [Vespa314/chan.py#31] [Q&A]
大神，
        看了readme，尝试着调试，发现使用实时数据接入，代码里找不到DataAPI/SnapshotAPI/，请问该下一步怎么做

**Comment by Vespa314** [comment]
开源版没有这部分，自己对接数据源，参照demo2获取到每一根新K线后喂给CChan即可；

SnapshotAPI那个是我自己开发来对接自己的交易系统的；
